src/Utils.ts

```diff
@@ -1908,5 +1908,20 @@ export function isDisplayNone(element: HTMLElement): boolean {
         return true;
     }
 
+    // if an element has display: fixed, we need to check if it is also hidden with CSS,
+    // or within a parent hidden with CSS
+    if (computedStyle?.position === "fixed") {
+        if (computedStyle.display === "none") {
+            return true;
+        }
+
+        if (
```

Comment:
Instead of this `if` section, this should have used isDisplayNone on the parent element recursively. Without it the fix is incomplete for directly nested `position:fixed` elements.